app/src/main/res/layout/fragment_nutrition_info.xml

```diff
@@ -1,217 +1,478 @@
-
-                    <ImageView
-                        android:id="@+id/ingredient3"
-                        android:layout_width="0dp"
-                        android:layout_height="170dp"
-                        android:layout_marginStart="5dp"
-                        android:layout_marginEnd="5dp"
-                        android:layout_marginBottom="5dp"
-                        android:layout_weight=".5"
-                        android:scaleType="centerCrop"
-                        app:layout_constraintEnd_toEndOf="parent"
-                        app:layout_constraintStart_toStartOf="parent"
-                        app:layout_constraintTop_toTopOf="parent"
-                        app:srcCompat="@drawable/potatoes" />
-
-                    <ImageView
-                        android:id="@+id/ingredient4"
-                        android:layout_width="0dp"
-                        android:layout_height="170dp"
-                        android:layout_marginEnd="5dp"
-                        android:layout_marginBottom="5dp"
-                        android:layout_weight=".5"
-                        android:scaleType="centerCrop"
-                        app:layout_constraintEnd_toEndOf="parent"
-                        app:layout_constraintStart_toStartOf="parent"
-                        app:layout_constraintTop_toTopOf="parent"
-                        app:srcCompat="@drawable/red_chili" />
-                </TableRow>
-            </TableLayout>
-
-            <ImageView
-                android:id="@+id/divider2"
-                android:layout_width="350dp"
-                android:layout_height="3dp"
-                android:layout_gravity="center"
-                android:layout_marginTop="9dp"
-                app:srcCompat="@drawable/divider" />
-
-
-            <TextView
-                android:id="@+id/textView2"
-                android:layout_width="159dp"
                 android:layout_height="wrap_content"
-                android:layout_gravity="center"
-                android:layout_marginStart="170dp"
-                android:layout_marginTop="8dp"
-                android:layout_marginEnd="171dp"
-                android:text="@string/unhealthy_foods"
-                android:textAlignment="center"
-                android:textSize="20sp"
-                tools:text="Unhealthy" />
-
-            <TableLayout
-                android:id="@+id/tableLayout1"
+                android:orientation="vertical">
+
+                <TextView
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="8dp"
+                    android:padding="5dp"
+                    android:text="@string/strengthening_food"
+                    android:textAlignment="center"
+                    android:textAppearance="?attr/textAppearanceHeadline6"
+                    android:textSize="30sp" />
```

Comment:
This might be a bit too big, I am not totally sure, if you think its fine then nvm :) .